src/marks/auto.js

```diff
@@ -91,7 +86,7 @@ export function auto(data, {x, y, color, size, fx, fy, mark} = {}) {
   // Determine the default mark type.
   if (mark === undefined) {
     mark =
-      sizeValue != null || sizeReduce != null
+      size != null || sizeReduce != null
```

Comment:
What is this change doing?

Reply:
At one point in refactoring, I didn't have a reference to sizeValue here, and it seemed inconsistent to refer to x and y (not xValue and yValue) below, but sizeValue (not size) here, so I changed it. Upon further reflection, if sizeValue is undefined / null, then size will be undefined / null, so it doesn't matter. I'll revert this.

Reply:
I think it’ll be clearer if we store the materialized columns as a separate array, rather than redefining local variables/function arguments. I’ll incorporate into this into a commit I’m about to push.